string anchored_system
string text
string text_name
float64[] relative_position
float64[] color_rgba
float64 font_size
string font